app-ios/tutanota/Sources/Alarms/AlarmModel.swift

```diff
@@ -2,235 +2,169 @@ import Foundation
-  let occurenceDate: Date
+	let occurrenceNumber: Int
+	let occurenceDate: Date
 }
 
 struct AlarmOccurence: Equatable {
-  let occurrenceNumber: Int
-  let eventOccurrenceTime: Date
-  let alarm: AlarmNotification
-
-  /// Calculates alarm occurence time for a given event occurnce time and trigger from alarm.
-  func alarmOccurenceTime() -> Date {
-    return AlarmModel.alarmTime(trigger: alarm.alarmInfo.trigger, eventTime: eventOccurrenceTime)
-  }
+	let occurrenceNumber: Int
+	let eventOccurrenceTime: Date
+	let alarm: AlarmNotification
+
+	/// Calculates alarm occurence time for a given event occurnce time and trigger from alarm.
+	func alarmOccurenceTime() -> Date { AlarmModel.alarmTime(trigger: alarm.alarmInfo.trigger, eventTime: eventOccurrenceTime) }
 }
 
 /// Something that can calculate when alarms should happen
 protocol AlarmCalculator {
-  /// Calcuate the soonest alarm occurences up to the specified limits
-  /// note: return type would ideally not be required to be boxed but it's the easiest until proper upper bound inference is available at runtime
-  /// see https://forums.swift.org/t/inferred-result-type-requires-explicit-coercion/59602/2
-  /// (alternatively we could always produce arrays)
-  func futureOccurrences(acrossAlarms alarms: [AlarmNotification], upToForEach: Int, upToOverall: Int) -> any BidirectionalCollection<AlarmOccurence>
-
-  /// Calculate upcoming alarm occurences for a single alarm
-  /// - Returns: lazy sequence of alarm occurences. It might be infinite if alarm repeats indefinitely!
-  func futureOccurrences(ofAlarm alarm: AlarmNotification) -> any Sequence<AlarmOccurence>
+	/// Calcuate the soonest alarm occurences up to the specified limits
+	/// note: return type would ideally not be required to be boxed but it's the easiest until proper upper bound inference is available at runtime
+	/// see https://forums.swift.org/t/inferred-result-type-requires-explicit-coercion/59602/2
+	/// (alternatively we could always produce arrays)
+	func futureOccurrences(acrossAlarms alarms: [AlarmNotification], upToForEach: Int, upToOverall: Int) -> any BidirectionalCollection<AlarmOccurence>
+
+	/// Calculate upcoming alarm occurences for a single alarm
+	/// - Returns: lazy sequence of alarm occurences. It might be infinite if alarm repeats indefinitely!
+	func futureOccurrences(ofAlarm alarm: AlarmNotification) -> any Sequence<AlarmOccurence>
 }
 
 /// A helper to magically unbox any Sequence to call prefix() on it because
 /// "we cannot call a member of an existential value if the result type of that member uses any of the associated types in an invariant position."
 /// see https://forums.swift.org/t/se-0353-constrained-existential-types/56853/22
-func prefix(_ s: some Sequence<AlarmOccurence>, _ maxLength: Int) -> any Sequence<AlarmOccurence> {
-  return s.prefix(maxLength)
-}
+func prefix(_ s: some Sequence<AlarmOccurence>, _ maxLength: Int) -> any Sequence<AlarmOccurence> { s.prefix(maxLength) }
```

Comment:
this might not be needed anymore, we should check  at some point

Reply:
If I undo this, it asks me to remove the lines.